core/cli/cli.js

```diff
@@ -118,3 +155,48 @@ function green_checkmark() {
     const chalk = require('chalk');
     return chalk.green('\u2714');
 }
+
+function createScaffold(projectName) {
+    const {homeViewTemplate, homePageTemplate} = require('./templates/homeTemplate');
+    const viewTemplate = homeViewTemplate();
+    const pageTemplate = homePageTemplate(projectName);
+    let currentDir = path.normalize(path.resolve(process.cwd(), projectName));
+
+    fs.mkdir(currentDir, (err) => {
+        if (err) {
+            console.log(err);
+        } else {
+            let tempPath = path.resolve(currentDir, 'app');
+            fs.mkdir(tempPath, (err) => {
+                if (err) {
+                    console.log(err);
+                } else {
+                    let viewPath = path.resolve(currentDir, 'app', 'views');
+                    fs.mkdir(viewPath, (err) => {
+                        if (err) {
+                            console.log(err);
+                        } else {
+                            let tempPath = path.resolve(currentDir, 'app', 'views', 'homeView.js');
+                            fs.writeFile(tempPath, viewTemplate, (err) => {
+                                if (err)
+                                    console.log(err);
+                            });
+                        }
+                    });
+                    let pagePath = path.resolve(currentDir, 'app', 'pages');
+                    fs.mkdir(pagePath, (err) => {
+                        if (err) {
+                            console.log(err);
+                        } else {
+                            let tempPath = path.resolve(currentDir, 'app', 'pages', 'homePage.js');
```

Comment:
It needs to be `homePage.html.js`. (I'm thinking of getting rid of the `.html.js` suffix requirement btw.)

Reply:
Sorry about that!  I completely forgot about the .html part.  I'm adding for now and if you decide to remove that in the future, we can adjust.